src/main/java/seedu/address/ui/NearbyPersonListPanel.java

```diff
@@ -69,6 +72,12 @@ private void scrollTo(int index) {
         });
     }
 
+    @Subscribe
+    private void handleJumpNearbyToListRequestEvent(JumpToNearbyListRequestEvent event) {
```

Comment:
Perhaps the function could be `handleJumpToNearbyListRequestEvent` for clarity

Reply:
good catch